nginx/ngx_js.c

```diff
@@ -3271,30 +3252,18 @@ ngx_js_preload_object(ngx_conf_t *cf, ngx_command_t *cmd, void *conf)
 }
 
 
-ngx_int_t
-ngx_js_init_preload_vm(ngx_conf_t *cf, ngx_js_loc_conf_t *conf)
+static ngx_int_t
+ngx_js_init_preload_vm(njs_vm_t *vm, ngx_js_loc_conf_t *conf)
 {
     u_char               *p, *start;
     size_t                size;
-    njs_vm_t             *vm;
     njs_int_t             ret;
     ngx_uint_t            i;
-    njs_vm_opt_t          options;
     njs_opaque_value_t    retval;
     ngx_js_named_path_t  *preload;
 
-    njs_vm_opt_init(&options);
-
-    options.init = 1;
-    options.addons = njs_js_addon_modules_shared;
-
-    vm = njs_vm_create(&options);
-    if (vm == NULL) {
-        goto error;
-    }
-
     njs_str_t str = njs_str(
-        "import fs from 'fs';"
+        "import __fs from 'fs';"
```

Comment:
Looks like no need rename fs, its scope is current module only.

Reply:
No, it is in the global scope.

if there is a js_import directive as follow

`js_import fs from lib.js` or `js_import fs.js`

we will get 
`SyntaxError: "fs" has already been declared `.